Sasquatch/SasquatchObjC/AppDelegate.m

```diff
@@ -104,11 +104,11 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
                        [[NSUserDefaults standardUserDefaults] removeObjectForKey:kMSStorageMaxSizeKey];
 
                        // Show alert.
-                       UIAlertController *alertController = [UIAlertController alertControllerWithTitle:@"Warning!"
-                                                                                                message:@"The maximum size of the internal "
-                                                                                                        @"storage could not be set."
+                       UIAlertController *alertController = [UIAlertController alertControllerWithTitle:NSLocalizedString(@"Warning!", nil)
+                                                                                                message:NSLocalizedString(@"The maximum size of the internal "
+                                                                                                        @"storage could not be set.", nil)
                                                                                          preferredStyle:UIAlertControllerStyleAlert];
-                       [alertController addAction:[UIAlertAction actionWithTitle:@"OK" style:UIAlertActionStyleDefault handler:nil]];
+                       [alertController addAction:[UIAlertAction actionWithTitle:NSLocalizedString(@"OK", nil) style:UIAlertActionStyleDefault handler:nil]];
```

Comment:
Is that right? First argument seems to be the __key__: https://developer.apple.com/documentation/foundation/nslocalizedstring

Reply:
I tested these changes. Crashed the sasquatch puppet and set a storage max size as a very large number. All alerts were shown as expected.